code/game/objects/items/weapons/storage/backpack.dm

```diff
@@ -38,6 +38,7 @@
 	max_w_class = WEIGHT_CLASS_HUGE
 	max_combined_w_class = 35
 	burn_state = FIRE_PROOF
+	cant_hold = list(/obj/item/weapon/storage/backpack/holding)

```

Comment:
Will this trigger before or after the proc that spawns the singularity?

Basically, this won't break using the GUI to bag of holding into bag of holding?